test-suite/tests/error-p005-context.jsonld

```diff
@@ -0,0 +1,5 @@
+{
+  "@context": {
+    "@version": 1.1
```

Comment:
Do we want this to be a number (float/double) or a string? Expressing version attributes as strings seems to be fairly common practice.